examples/Complex-Sounding-Plot.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (c) 2015,2016,2017 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+=================
```

Comment:
Add more equal signs to be the same length as the title. That will quash the errors for the doc creation.

Reply:
Ah okay easy enough, on it. Thanks!